robottelo/hosts.py

```diff
@@ -504,6 +505,10 @@ def install_katello_ca(self, satellite):
         :raises robottelo.hosts.ContentHostError: If katello-ca wasn't
             installed.
         """
+        warnings.warn(
+            message='The install_katello_ca method is deprecated, use the register method instead.',
+            category=DeprecationWarning,
```

Comment:
Can you add similar warning to install_capsule_katello_ca func?

Reply:
Good catch. Updated.